### ISSUE TITLE
[REQUEST] Allow for double input - specifically COD:WZ

### ISSUE THREAD

**Issue by astra-io** [Ryochan7/DS4Windows#1993]
I don't know if anyone's thought of this, but I want to use the native DS4 controller sticks, and a few macros from the emulated ds4win. You can't use any button mappings on the DS4Win emulation when the native DS4 is active, which is what I require.

I managed to get this working VERY briefly, where I disabled the sticks on DS4, but kept it through the native support, and kept all my button mappings and macros as well. I lost it very quickly, and haven't been able to reproduce it.

Does anyone know how I can make this possible?? Wouldn't this be a good solution for a lot of people, who want that natural feel + DS4 featuers?

**Comment by mika-n** [comment]
what kind of macros you use? If those macros send out only kbd and mouse events then you could tick "Use DInput Only" option in a profile. This way the original ds4 pad takes care of gamepad events and DS4Windows app takes care of sending out kbd/mouse macro events.

But if a macro sends out gamepad button events then the only way to do it is the use of virtual output gamepad device through ViGem driver. And this virtual output driver will be a new gamepad device. However, you can tweak the default profile and remove all default mappings so the virtual gamepad doesn't send duplicate btn events unless you press some macro trigger.

**Comment by Ryochan7** [comment]
It would probably just be easier to reset the default XInput focused profile settings in the profile and use base settings instead. Use Dead zone 0.00, Anti-dead zone 0.00, Max zone 1.00.

https://github.com/Ryochan7/DS4Windows/wiki/Settings#axis-zone-mapping

**Comment by astra-io** [comment]
Hmm.. so I want to stick to DInput as I play competitively - I've been playing on ds4w emulated for awhile and it's been fairly inconsistent to me, while feeling very responsive. However, the other day I had unknowingly played without ds4w being active, and I immediately knew something was different - that it was slower. I planned on changing it right away after I finished, but I ended up playing the best I had in months, beating my PR in kills. Although it felt "slower" my accuracy and control were immediately better, with no warm up or practice. 

It was on default settings before that (0 dz, 0 adz)

I've switched to rewasd and it seems to do everything I need, even though the dinput is still emulated. They claim it's 1:1 but even that feels like 90%.

Anyways, thanks for all the support!
